Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1166,10 +1177,44 @@ WarpXOpenPMDPlot::GetMeshCompNames (int meshLevel,
     }
 
     if ( 0 == meshLevel )
-      return;
+        return;
 
     field_name += std::string("_lvl").append(std::to_string(meshLevel));
 }
+/* Split varname into name and theta mode mode index
+ *
```

Comment:
Maybe add a few example inputs -> out, puts after a newline in the body of this function to make it easier to understand what goes in and out in practice :)
```suggestion
 *
 * Examples:
 *   bla_bli_imag_2 -> bla_bli, 5
 *   bla_bli_real_1 -> bla_bli, 2
```